runtime/Makefile.cmake.bitcode

```diff
@@ -17,4 +17,8 @@ ifneq ($(ENABLE_POSIX_RUNTIME),0)
 	DIRS += POSIX
 endif
 
+ifneq ($(BUILD_KLEE_EH_CXX),0)
```

Comment:
You need to add this to `Makefile.cmake.bitcode.config.in`, otherwise it will just evaluate to "".

Reply:
Hi, thanks for the comment! I have a set of patches by @jbuening lying around (thanks!), which fix this issue and improve build support in general, I'll try to make some time to push them soon.

Reply:
Pushed them.